src/mahoji/lib/abstracted_commands/duelCommand.ts

```diff
@@ -151,8 +167,19 @@ export async function duelCommand(
 		if (selection.customID === 'CONFIRM') {
 			return await confirm(amount);
 		}
-	} catch (err) {
-		return cancel();
+	} catch (err: any) {
+		const code = typeof err === 'object' && 'code' in err ? err.code : null;
+		if (code === 'INTERACTION_COLLECTOR_ERROR') {
+			return cancel();
```

Comment:
obviously we don't want to log an error if 'timeout' is the reason, but if it's another error/reason/message then we do want to log that.